test/unit/cookie.test.js

```diff
@@ -193,5 +221,178 @@ describe('Cookie', function () {
+                    secure: false,
+                    session: false,
+                    value: 'fooTest'
+                },
+                unparsedSingle = Cookie.unparseSingle(new Cookie(rawCookie));
+            expect(unparsedSingle).to.equal(rawCookie.name + '=' + rawCookie.value);
+        });
+
+        it('should return only the value in case of no cookie name specified', function () {
+            var rawCookie = {
+                    expires: 1502442248,
+                    hostOnly: false,
+                    httpOnly: false,
+                    path: '/',
+                    secure: false,
+                    session: false,
+                    value: 'bar'
+                },
+                unparsedSingle = Cookie.unparseSingle(new Cookie(rawCookie));
+            expect(unparsedSingle).to.equal('bar');
+        });
+    });
+
+    describe('unparse', function () {
+        it('should merge multiple cookies into a single string', function () {
+            var rawCookie1 = {
+                    name: 'testCookie1',
+                    domain: '.httpbin.org',
+                    expires: 1502442248,
+                    path: '/',
+                    value: 'fooTest1'
+                },
+                rawCookie2 = {
+                    name: 'testCookie2',
+                    domain: '.httpbin.org',
+                    expires: 1502442248,
+                    path: '/',
+                    value: 'fooTest2'
+                },
+                cookieArray = [],
+                unparse;
+            cookieArray.push(new Cookie(rawCookie1));
+            cookieArray.push(new Cookie(rawCookie2));
+
+            unparse = Cookie.unparse(cookieArray);
+            expect(unparse).to.equal('testCookie1=fooTest1; testCookie2=fooTest2');
+        });
+    });
+
+    describe('stringify', function () {
+        it('should return single Set-Cookie header string with key value', function () {
+            var cookie = new Cookie({
+                name: 'foo',
+                value: 'fooTest'
+            });
+
+            expect(Cookie.stringify(cookie))
+                .equals('foo=fooTest');
+        });
+
+        it('should return single Set-Cookie header string with expires', function () {
+            var cookie = new Cookie({
+                name: 'foo',
+                value: 'fooTest',
+                expires: '14 Jun 2017 00:00:00 PDT'
+            });
+
+            expect(Cookie.stringify(cookie))
+                .equals('foo=fooTest; Expires=Wed, 14 Jun 2017 07:00:00 GMT');
+        });
+
+        it('should return single Set-Cookie header string with secure and httpOnly set to true', function () {
+            var cookie = new Cookie({
+                name: 'foo',
+                value: 'fooTest',
+                httpOnly: true,
+                secure: true
+            });
+
+            expect(Cookie.stringify(cookie))
+                .equals('foo=fooTest; Secure; HttpOnly');
```

Comment:
Split into multiple lines only if it exceeds 120 chars limit.

Reply:
Fixed.